public/app/features/alerting/unified/components/admin/AlertmanagerConfig.test.tsx

```diff
@@ -107,37 +107,29 @@ describe('Admin config', () => {
     expect(ui.confirmButton.query()).not.toBeInTheDocument();
   });
 
-  it('Edit and save alertmanager config', async () => {
+  it('Editable alertmanager config', async () => {
     let savedConfig: AlertManagerCortexConfig | undefined = undefined;
 
     const defaultConfig = {
-      template_files: {
-        foo: 'bar',
+      template_files: {},
+      alertmanager_config: {
+        route: {
+          receiver: 'old one',
+        },
       },
-      alertmanager_config: {},
-    };
-
-    const newConfig = {
-      template_files: {
-        bar: 'baz',
-      },
-      alertmanager_config: {},
     };
 
     mocks.api.fetchConfig.mockImplementation(() => Promise.resolve(savedConfig ?? defaultConfig));
     mocks.api.updateAlertManagerConfig.mockResolvedValue();
     renderAdminPage(dataSources.alertManager.name);
-    const input = await ui.configInput.find();
-    expect(input.value).toEqual(JSON.stringify(defaultConfig, null, 2));
-    await userEvent.clear(input);
-    // What is this regex replace doing? in userEvent v13, '{' and '[' are special characters.
-    // To get the literal character, you have to escape them by typing '{{' or '[['.
-    // See https://github.com/testing-library/user-event/issues/584.
-    await userEvent.type(input, JSON.stringify(newConfig, null, 2).replace(/[{[]/g, '$&$&'));
```

Comment:
No longer a need for this line? No need to test if the value was updated?

Reply:
So this line doesn't actually work, a large part of this test assumed the input was a `textarea` field – but Monaco is a lot complexer than that and I couldn't make it focus the editor, clear it and type a new payload :( 

The hardest part was clearing the editor since it didn't accept a `{clearall}` command or `{meta}a{/meta} {backspace}`.